src/gui/TimeLineWidget.cpp

```diff
@@ -329,64 +329,18 @@ void TimeLineWidget::contextMenuEvent(QContextMenuEvent*)
 
 void TimeLineWidget::mousePressEvent( QMouseEvent* event )
 {
-	// TODO: Read these from a config
```

Comment:
Can we perhaps leave these in? Except they'd now be in the select action method, that is.

Reply:
The comments? They are.

Reply:
The variables holding action enums.

Reply:
Well, alright

Reply:
Now I'm thinking of ways to solve the problem I've created:
1. Fix this in #11, ask you to bear with me, and not do this again
2. `git rebase; git push --force` while laughing hysterically

Reply:
Doesn't option one rely on #9 and #10 going through unmodified? Or at least, won't we have this same issue again? I remake my fork from scratch every now and then so we can pollute it with PRs if that helps keep the history on https://github.com/LMMS/lmms/pull/6021 cleaner.

Reply:
You're right, I'm just lazy. Is it ever ok to `push --force` on a PR?

Reply:
Not sure, it's been long enough since my last horrible git screwup that I've forgotten what is and isn't dangerous. I think it should be fine as long as whatever you force on is acceptable as the new history? So e.g. if you want to squash everything I don't think it should matter.

It might screw up review comments though, IIRC they can get orphaned.